src/plugins/navigation-data/index.js

```diff
@@ -0,0 +1,53 @@
+(function() {
+  function Laser(name, deps) {
+    console.log('Navigation Data plugin loaded');
+
+    var navdata = {
+      roll: 0,
+      pitch: 0,
+      yaw: 0,
+      thrust: 0,
+      depth: 0,
+      heading: 0
+    };
+
+    // Arduino
+    deps.rov.on('status', function (status) {
+      if ('hdgd' in status) {
+        navdata.heading = status.hdgd;
+      }
+      if ('deap' in status) {
+        navdata.depth = status.deap;
+      }
+      if ('pitc' in status) {
+        navdata.pitch = status.pitc;
+      }
+      if ('roll' in status) {
+        navdata.roll = status.roll;
+      }
+      if ('yaw' in status) {
+        navdata.yaw = status.yaw;
+      }
+      if ('fthr' in status) {
+        navdata.thrust = status.fthr;
+      }
+      if ('mtrmod' in status){
```

Comment:
mtrmod is not a navdata construct but instead related to moto configurations... so mabey motodiag